src/options/options.css

```diff
@@ -78,6 +78,14 @@ input[type=radio] ~ .helper-text {
 #popupIconColor {
   display: inline;
 }
+
+#qrQuietZone {
+  position: relative;
+  top: 5px;
+  cursor: pointer;
+  -webkit-appearance: none;
```

Comment:
Why that though? In my case it looks a little better without that overriding:

Without:
![image](https://user-images.githubusercontent.com/11966684/47821132-f6f9d580-dd5f-11e8-9b57-5c6d66458596.png)

With:
![image](https://user-images.githubusercontent.com/11966684/47821153-0416c480-dd60-11e8-9f0d-cc0b0236b804.png)


(In GNOME here)

Reply:
(I've changed it for now for merging into master. However, feel free to point out why this may be needed on some platforms or so…)